cvat/settings/base.py

```diff
@@ -514,9 +514,7 @@ class CVAT_QUEUES(Enum):
 DATA_UPLOAD_MAX_MEMORY_SIZE = 100 * 1024 * 1024  # 100 MB
 DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
 DATA_UPLOAD_MAX_NUMBER_FILES = None
-LOCAL_LOAD_MAX_FILES_COUNT = 500
 LOCAL_LOAD_MAX_FILES_SIZE = 512 * 1024 * 1024  # 512 MB
```

Comment:
What about `LOCAL_LOAD_MAX_FILES_SIZE` variable? It doesn't seem to be used anywhere. Should we delete it too?

Reply:
Removed